kubernetes/k8s_workflow_integration.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Serverless Workflow Specification Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"github.com/serverlessworkflow/sdk-go/v2/model"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// This package provides a very simple api for kubernetes operator to test the integration
+// of the Serverless SDK-Go with operator-sdk controller-gen and deepcopy-gen tools.
+// The purpose of this integration is to stop issues like below beforehand:
+// github.com/serverlessworkflow/sdk-go/model/event.go:51:2: encountered struct field "" without JSON tag in type "Event"
+// github.com/serverlessworkflow/sdk-go/model/states.go:66:12: unsupported AST kind *ast.InterfaceType
+
+// SDKServerlessWorkflowSpec defines a base API for integration test with operator-sdk
+type SDKServerlessWorkflowSpec struct {
```

Comment:
```suggestion
type ServerlessWorkflowSpec struct {
```
Fits better, don't you think? 😎

Reply:
sounds good.